schema_salad/__init__.py

```diff
@@ -11,5 +12,15 @@
 
 import six
 if six.PY3:
+
+    if onWindows:
+        # create '/tmp' folder if not present
+        # required by autotranslate module
+        if not os.path.exists("/tmp"):
+            try:
+                os.mkdirs("/tmp")
```

Comment:
@kapilkd13 Does this mean we're checking and creating in the current directory or ``Drive:\ root``? Also, can you test this code?